resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -323,6 +339,16 @@ class VisualEffect
             {
                 $transparencyValue = if ($this.TransparencyEffects) { 0 } else { 1 }
 				
+                if (-not (DoesRegistryKeyPropertyExist -Path $global:PersonalizationRegistryPath -Name ([VisualEffect]::TransparencySettingProperty))) {
+                    New-ItemProperty -Path $global:PersonalizationRegistryPath -Name ([VisualEffect]::TransparencySettingProperty) -Value $transparencyValue -PropertyType DWord
+                }
+                Set-ItemProperty -Path $global:PersonalizationRegistryPath -Name ([VisualEffect]::TransparencySettingProperty) -Value $transparencyValue 
+            }

```

Comment:
I've noticed you've copied this snippet every time we add a new setting. [Set-ItemProperty](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/set-itemproperty?view=powershell-7.4) creates the value for you if it doesn't exist so no need to call `New-ItemProperty` here.

Reply:
TransparencyEffects needs it to specify that the key is a `DWORD`. But you're right, including it here was an error.